lms/templates/courseware/progress.html

```diff
@@ -198,7 +198,7 @@ <h4 class="hd hd-4">
                                         <p class="override-notice">
                                             %if section.override is not None:
                                                 <%last_override_history = section.override.get_history().order_by('created').last()%>
-                                                %if (not last_override_history or last_override_history.feature == grades_constants.GradeOverrideFeatureEnum.proctoring) and section.format == "Exam" and earned == 0:
+                                                %if (not last_override_history or last_override_history.system == grades_constants.GradeOverrideFeatureEnum.proctoring) and section.format == "Exam" and earned == 0:
```

Comment:
Could you explain a bit why changing from `.feature` to `.system` fix the error?

Reply:
The manual history table has a column called feature. In the simple history, this column is called system.